spec/features/create_todo_items_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe "Todo items" do
+   before do
+      @user = create(:user)
+      visit(user_path(@user))
+    end
+  
```

Comment:
I don't know how valid a test like this is since I am not using the factory generated todo_item, although, I do see some validity in testing whether the database is registering the newly created todo_item